doc/authentication-backends/LDAP-authentication-module.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+An LDAP authentication module.
+It provides read-only abstraction over LDAP directory.
+User credentials are verified by performing LDAP bind with username and password provided by a client.
+
+Requires SASL PLAIN method.
+
+## Configuration options
+
+* **ldap_base:**
+    * **Description:**  LDAP base directory which stores users accounts.
+    * **Values:** String
+    * **Default:** This option is required
+
+* **ldap_uids:**
+    * **Description:**  LDAP attribute which holds a list of attributes to use as alternatives for getting the JID.
+   The attributes are of the form: `[{ldap_uidattr}]` or `[{ldap_uidattr, ldap_uidattr_format}]`. You can use as many comma separated attributes as needed.
+    * **Values** `[ ldap_uidattr | {ldap_uidattr: ldap_uidattr_format} ]`
+    The values for `ldap_uidattr` and `ldap_uidattr_format` are described as follow:
+         * **ldap_uidattr:** LDAP attribute which holds the user’s part of a JID. The default value is `uid`
+         * **ldap_uidattr_format:**  Format of the `ldap_uidattr` variable. The format must contain one and only one pattern variable `%u` which will be replaced by the user’s part of a JID. For example, `%u@example.org`. The default value is `%u`.
+    * **Default**  `[{uid, %u}]`
+
+* **ldap_filter:**
+    * **Description:** LDAP filter. Please, do not forget to close brackets and do not use superfluous whitespaces.
```

Comment:
an LDAP filter. 
Please, do not forget to close the brackets and do not use superfluous whitespaces.